app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
```suggestion
class NotWearingMaskError(Exception):
```
All user-defined exceptions should be derived from Exception class. https://docs.python.org/3/library/exceptions.html#Exception